plugin/src/main/java/net/dzikoysk/funnyguilds/feature/hooks/HookManager.java

```diff
@@ -74,7 +75,14 @@ public void setupHooks() {
                 .subscribe(hook -> PLACEHOLDER_API = hook);
 
         this.<HologramsHook>setupHook("HolographicDisplays", true, pluginName -> new HolographicDisplaysHook(pluginName, this.plugin), true)
-                .subscribe(hook -> HOLOGRAMS = hook);
+                .subscribe(hdHook -> {
+                    if (hdHook.isEmpty()) {
```

Comment:
Ja bym to nadal rozbił na dwa oddzielne setupHook(), jeśli użytkownik FG chce mieć na serwerze dwa pluginy na to - niech sobie w configu wyłączy jedną z integracji, o ile w ogóle by zauważył jakieś różnice. A nam w kodzie nie robi różnicy, że jeden hook za chwilę potencjalnie nadpisze drugi, a nie będziemy ich tak mieszać tutaj w jednym setupie.